### PR TITLE
ci: automate snapshot updates

### PR DESCRIPTION
**Pull request by Ad96el**
## fixes KILTProtocol/ticket#3408

According to the [Acala approach](https://github.com/AcalaNetwork/e2e-tests/blob/master/.github/workflows/update.yml), this pull request introduces a new GitHub workflow to automatically update the snapshots. The workflow must be triggered manually, resulting in a pull request that can then be approved by us. 

Additionally, this PR introduces the currently updated snapshots.

An example PR can be found here: https://github.com/KILTprotocol/kilt-node/pull/638

### CODE REVIEW

File: .github/workflows/update-snapshots.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+env:
+  GH_TOKEN: ${{ github.token }}
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        submodules: recursive
```
Comment (ntn-x2) [inline .github/workflows/update-snapshots.yaml:18]: We don't have submodules, we can get rid of this.

File: .github/workflows/update-snapshots.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - name: Setup yarn
+      run: |
+        cd ./integration-tests/chopsticks
```
Comment (ntn-x2) [inline .github/workflows/update-snapshots.yaml:21]: Please add a comment that this is copy-pasted from our other CI step, so we can link the two together.

File: .github/workflows/update-snapshots.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        yarn test:CI -u
+    - name: Commit and Create PR
+      uses: actions/github-script@v6
+      with:
+        script: |
+          const branchName = `update-snapshots-${context.sha.slice(0, 7)}`
+          await exec.exec(`git config user.email "devs@kilt.io"`)
+          await exec.exec(`git config user.name "kiltbot"`)
+          await exec.exec(`git checkout -b ${branchName}`)
+          await exec.exec(`git`, ['commit', '-am', 'ci: Update snapshots'])
```
Comment (ntn-x2) [inline .github/workflows/update-snapshots.yaml:41]: If we take any actions based on https://github.com/KILTprotocol/kilt-node/pull/639#discussion_r1604375370, then this should be updated to reflect that.

**Review by ntn-x2 — APPROVED** [review]
Let's see how it works 👍👍👍

File: .github/workflows/update-snapshots.yaml (hunk 1 above)
Comment (Ad96el) [reply .github/workflows/update-snapshots.yaml]: Updated in: [b57b55d](https://github.com/KILTprotocol/kilt-node/pull/636/commits/b57b55dd1b29ae84b7ad429de93691f4e867ea7a)

File: .github/workflows/update-snapshots.yaml (hunk 2 above)
Comment (Ad96el) [reply .github/workflows/update-snapshots.yaml:19]: Comment added in: [b57b55d](https://github.com/KILTprotocol/kilt-node/pull/636/commits/b57b55dd1b29ae84b7ad429de93691f4e867ea7a)

File: .github/workflows/update-snapshots.yaml (hunk 3 above)
Comment (Ad96el) [reply .github/workflows/update-snapshots.yaml:39]: I added the label on hold until we found a solution/decision for the other PR

**Comment by ntn-x2** [comment]
Closing this for now as it's probably outdated by https://github.com/KILTprotocol/ticket/issues/3252. If needed, we'll opened a new our from `develop` directly.

**Pull request closed without merging**